lighthouse-extension/test/extension-test.js

```diff
@@ -106,7 +106,7 @@ describe('Lighthouse chrome extension', function() {
 
   const selectors = {
     audits: '.lh-audit,.lh-perf-metric,.lh-perf-hint',
-    titles: '.lh-score__title, .lh-perf-hint__title, .lh-perf-metric__title',
```

Comment:
I believe you'll have to update viewer test too?

Reply:
yup good catch